app/src/main/java/io/fia/rememberall/data/Task.java

```diff
@@ -4,8 +4,10 @@
 import java.text.SimpleDateFormat;
 import java.util.Calendar;
 import java.util.Date;
+import java.util.Objects;
 
 public class Task {
+    int ID;
```

Comment:
Is this field used anywhere?

Reply:
Nope. I was thinking we might want to guarantee uniqueness and possibly use it in the hashcode function, but I didn’t end up using it in this version.